content/en/docs/concepts/containers/_index.md

```diff
@@ -1,5 +1,44 @@
 ---
-title: "Containers"
+title: Containers
 weight: 40
+description: Technology for packaging the (compiled) code for an application along with the dependencies it needs at run time.
```

Comment:
To shorten:
Technology for packaging an application along with its runtime dependencies.